hack/update-protobuf.sh

```
@@ -2,50 +2,11 @@
-
-PATH="$PATH:${protoc_bin_dir}" "${GO_TO_PROTOBUF}" \
-  --output-dir="${GOPATH}/src" \
-  --apimachinery-packages='-k8s.io/apimachinery/pkg/util/intstr,-k8s.io/apimachinery/pkg/api/resource,-k8s.io/apimachinery/pkg/runtime/schema,-k8s.io/apimachinery/pkg/runtime,-k8s.io/apimachinery/pkg/apis/meta/v1,-k8s.io/apimachinery/pkg/apis/meta/v1beta1,-k8s.io/api/core/v1,-k8s.io/api/rbac/v1' \
-  --go-header-file=${SCRIPT_ROOT}/hack/empty.txt \
-  --proto-import=${SCRIPT_ROOT}/third_party/protobuf \
-  --proto-import=${SCRIPT_ROOT}/vendor \
-  --proto-import=${SCRIPT_ROOT}/tools/vendor \
-  --packages="${API_PACKAGES}"
+# Add the protoc-gen-gogo directory to the PATH so that the generator can find it.
+PROTOC_GEN_GOGO_DIR="$(dirname "${PROTOC_GEN_GOGO}")"
+PATH="${PROTOC_GEN_GOGO_DIR}:${PATH}" GENERATOR=go-to-protobuf ${SCRIPT_ROOT}/hack/update-codegen.sh
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)